test/api/core_test.dart

```diff
@@ -85,12 +84,12 @@ void main() {
           ..url.asString.equals('${eg.realmUrl.origin}/api/v1/example/route')
           ..headers.deepEquals(authHeader(email: eg.selfAccount.email, apiKey: eg.selfAccount.apiKey))
           ..fields.deepEquals({})
-          ..files.single.which(it()
+          ..files.single.which((it) => it
             ..field.equals('file')
             ..length.equals(length)
             ..filename.equals(filename)
             ..has<Future<List<int>>>((f) => f.finalize().toBytes(), 'contents')
-              .completes(it()..deepEquals(content.expand((l) => l)))
+              .completes((it) => it..deepEquals(content.expand((l) => l)))
```

Comment:
nit:
```suggestion
              .completes((it) => it.deepEquals(content.expand((l) => l)))
```

Similarly in a few other places — wherever there's just one `..` cascading call.